docs/technologies/node.md

```diff
@@ -148,11 +148,11 @@ cat > /home/damner/code/.labtests/package.json << EOF
 EOF
 
 # run test
-(yarn vitest run --config=/home/damner/code/.labtests/config.js --threads=false --reporter=json --outputFile=/home/damner/code/.labtests/payload.json || true)  | tee /home/damner/code/.labtests/evaluationscript.log
+(bun vitest run --config=/home/damner/code/.labtests/config.js --threads=false --reporter=json --outputFile=/home/damner/code/.labtests/payload.json || true)  | tee /home/damner/code/.labtests/evaluationscript.log
 
 # Write results to UNIT_TEST_OUTPUT_FILE to communicate to frontend
 cd /home/damner/code/.labtests
-node process.js
+bun process.js
```

Comment:
Do not change the runtime to bun just yet.

Reply:
I've gotten a CJS/ESM error. So changed this to bun as well and fixed the issue